### ISSUE TITLE
HUSKYLENS.h contains reference to SoftwareSerial.h that won't run on all boards

### ISSUE THREAD

**Issue by jstanle1** [HuskyLens/HUSKYLENSArduino#5]
The SoftwareSerial library won't build or run on a lot of boards (eg., Adafruit ItsyBitsy M0 Express).  Including SoftwareSerial.h in HUSKYLENS.h breaks the HuskyLens library for those boards.

I think it would be better to remove SoftwareSerial.h from the library header and include it directly in the example code, possibly with a comment that it should be removed or commented out if it doesn't fit a given board (I just use Serial1 for UART, or remove it entirely for use with I2C).

**Comment by Robert-Proaps** [comment]
It appears that the library does not function without it.

**Comment by h26k2** [comment]
> It appears that the library does not function without it.

Yes, you are right as per my knowledge. I guess that it only functions on AVR Boards like { Arduino UNO, MEGA etc }
